util/buildRelease/install.sh

```diff
@@ -317,6 +317,19 @@ done
 # copy utf8-decoder header
 myinstallfile third-party/utf8-decoder/utf8-decoder.h "$DEST_THIRD_PARTY"/utf8-decoder/
 
+# copy mason
+if [ -f tools/mason/mason ]
+then
+  if [ ! -z "$PREFIX" ]
+  then
+    myinstallfile tools/mason/mason "$PREFIX/share/chapel/$VERS/tools/mason"
+    ln -s "$PREFIX/share/chapel/$VERS/tools/mason/mason" "$PREFIX/bin/mason"
```

Comment:
Is there some reason we can't actually but the main version in $PREFIX/bin/mason ? Is there something depending on the version in the tools directory? (If so, maybe it needs to be updated?).

I'm pretty sure that Debian Filesystem Hierarchy Standard rules would have us putting such binaries in to /usr/bin or maybe to /usr/lib. /usr/share is for processor-independent files.

Reply:
I've done the symbolic link because the Mason makefile does it too. I'll explore if it actually needs to stay in the tool section.

Reply:
@panzone: As we've been writing up our 1.16 release notes, I only just learned that the $CHPL_HOME/bin/ copy of mason is a symbolic link, as you say here, and wondered why, but don't think I've heard the answer yet.  Can one of @ben-albrecht / @benharsh / @spartee explain the rationale?  Thanks.

Reply:
I think we used a symbolic link because that's what c2chapel did/does. I don't think there's a deep reason why it couldn't be changed.

Reply:
Intuitively, I'd think in either a $CHPL_HOME-based build or a make install-style setting, we'd want to copy those binaries such that the original source tree could be clobbered / removed without breaking the tool...?

Reply:
The file is copied to an installation directory in the original version of this PR already. I was just bringing up that Debian won't like it if we put a binary in /usr/share.